src/v/archival/ntp_archiver_service.cc

```diff
@@ -359,14 +359,15 @@ ss::future<ntp_archiver::batch_result> ntp_archiver::wait_all_scheduled_uploads(
               _manifest.get_manifest_path());
         }
 
-        // TODO: error handling
         if (_partition->archival_meta_stm()) {
             retry_chain_node rc_node(
               _manifest_upload_timeout, _initial_backoff, &parent);
             auto result
```

Comment:
I think it is better to rename `result` to `error` or `ec` to avoid future confusion.

Reply:
Sounds good!